test/gtest/uct/ib/test_cq_moderation.cc

```diff
@@ -12,13 +12,12 @@ extern "C" {
 #include <poll.h>
 #include <infiniband/verbs.h>
 
-static const unsigned nsec_per_usec = (UCS_NSEC_PER_SEC / UCS_USEC_PER_SEC);
-
-/* wait for 3 usecs to get statistics */
+/* wait for 3 secs to get statistics */
```

Comment:
maybe 1 sec is enough? to reduce testing time

Reply:
updated to 1 sec